common/snapshots.py

```diff
@@ -1405,22 +1405,25 @@ def smartRemove(self, del_snapshots, log = None):
             additionalChars = len(self.config.sshPrefixCmd(cmd_type = str))
 
             head = 'screen -d -m bash -c "('
-            head += 'TMP=\$(mktemp -d); '                      #create temp dir used for delete files with rsync
-            head += 'test -z \\\"\$TMP\\\" && exit 1; '        #make sure $TMP dir was created
-            head += 'test -n \\\"\$(ls \$TMP)\\\" && exit 1; ' #make sure $TMP is empty
+            # create temp dir used for delete files with rsync
+            head += 'TMP=\\$(mktemp -d); '
+            # make sure $TMP dir was created
+            head += 'test -z \\\"\\$TMP\\\" && exit 1; '
+            # make sure $TMP is empty
+            head += 'test -n \\\"\\$(ls \\$TMP)\\\" && exit 1; '
```

Comment:
Sorry it is hard to see here.
In all three lines the `\$` was modified to `\\$`.